ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1763,3 +1763,73 @@ periodics:
         - key: sa.config-updater.arm01.config
           path: kubeconfig
         secretName: build-farm-credentials
+- agent: kubernetes
+  cluster: app.ci
+  decorate: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  interval: 12h
+  labels:
+    ci.openshift.io/role: infra
```

Comment:
Shouldn't it also be rehearsable?

Reply:
The rehearse will actually accept the invitations. For now, the rehearse tool is not intelligent enough to rehearse a job with extra/changed parameters (in this case `dry-run=true`).

Reply:
Yeah, what's the problem with taht?

Reply:
updated